pages/_app.tsx

```diff
@@ -7,7 +9,14 @@ import { useRouterScroll } from '@/hooks/useRouterScroll'
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 function MyApp({ Component, pageProps }: AppProps) {
   useRouterScroll()
-  return <Component {...pageProps} />
+  return (
+    <>
+      <GoogleTagManager></GoogleTagManager>
```

Comment:
他に合わせてこっちの方が見通しがいいかも
（lintで明記してないので強制じゃないです）

```suggestion
      <GoogleTagManager />
```